.snippets/code/products/settlement/get-started/snippet-2.ts

```diff
@@ -69,14 +76,14 @@ import { getSigner } from './helpers';
     tr,
     sender.signer,
     quote,
-    receiver.address
+    destAddress
   );
   console.log('Initiated transfer with receipt: ', receipt);
 
   await routes.checkAndCompleteTransfer(
     bestRoute,
     receipt,
-    receiver.signer,
+    undefined,
     15 * 60 * 1000
```

Comment:
```suggestion
    timeout
```
We could create a constant called 'timeout' and then input it here. Just to be more clear on what this line means